application-templates/starter/src/components/entry-point/entry-point.js

```diff
@@ -1,17 +1,13 @@
 import React from 'react';
+import { Redirect, Route, Switch } from 'react-router-dom';
 import {
   ApplicationShell,
   setupGlobalErrorListener,
   RouteCatchAll,
 } from '@commercetools-frontend/application-shell';
 import { Sdk } from '@commercetools-frontend/sdk';
 import * as globalActions from '@commercetools-frontend/actions-global';
-import { Redirect, Route, Switch } from 'react-router-dom';
-
-const loadApplicationMessagesForLanguage = async lang => {
-  const messages = await import(`../../i18n/data/${lang}.json` /* webpackChunkName: "application-messages-[request]" */);
```

Comment:
FYI: this causes webpack now to throw (no idea why it didn't before)

```bash
Failed to compile.
./src/components/entry-point/entry-point.js 53:19-55:13
Critical dependency: the request of a dependency is an expression
 @ ./src/components/entry-point/index.js
 @ ./src/index.js
 @ multi ../packages/mc-scripts/config/polyfills.js ../packages/mc-scripts/node_modules/core-js/stable/index.js ../packages/mc-scripts/node_modules/regenerator-runtime/runtime.js ./src/index.js
```

Reply:
I fixed this on my branch too. Did you look at the change set there?

Reply:
I know, I took it from your branch 😅

Reply:
Ah kay. It sounded like thought you found out something novel 😄.

Reply:
Yes sorry, I'm just "documenting" what I find along the way for reference, even though it might be obvious coming from the other PR.